frontend/src/components/cfp-form/index.tsx

```diff
@@ -116,28 +116,6 @@ export const CfpForm: React.SFC<Props> = ({
     variables: {
       conference: conferenceCode,
     },
-    onCompleted(data) {
```

Comment:
not 100% sure why this stopped working.

The same code works in the profile edit 🤷 

My hypothesis is that Apollo onCompleted flow changed somehow